bin/improver

```diff
@@ -77,4 +77,5 @@ fi
 export PYTHONPATH="$IMPROVER_DIR/lib/:${PYTHONPATH:-}"
 export PATH="$IMPROVER_DIR/bin/:$PATH"
 
-exec improver-$OPER "$@"
+# Run the IMPROVER app, wrapped with time to give resource usage information.
```

Comment:
As per discussion in the demo, I'd prefer this to be optional at the improver level and then made default within the suite with a flag in the rose-suite.conf. This makes turning it on and off a decision at the time of running the suite, and means we can turn it on by default in the trunk of the suite; we'll still get the output most of the time unless someone chooses to turn it off, e.g. when running operationally.

```
if [[ $1 == profile]] || [[ $1 == --profile ]]; then
    exec /usr/bin/time -v improver-$OPER "$@"
else
    exec improver-$OPER "$@"
fi
```

Reply:
Yeah, that sounds OK.
Though I'd like to make it the default, as most the time people will want to know what the resource usage of a trial, or what difference a change made without knowing beforehand. 
/usr/bin/time has very little overhead, so like the PBS epilogue, it's just useful info when you need it.

Reply:
Yes, we can have it on (by default) at the suite level, rather than the code level.

Reply:
We could be even more generic and allow any wrapper command:
```sh
exec ${IMPROVER_WRAPPER_CMD:-} improver-$OPER "$@"
```

This could be used to additionally add prolog/epilog commands around improver execution, e.g., Aaron's CDL generation in the suite could use that.

Not sure I would make it part of the improver command, though, as we're "profiling" from outside rather than inside the process. Feels a bit hacky. Alternatively, we could have a profiling wrapper command and insert it where needed (suite level and perhaps cli tests are likely the only places we need this).